tests/e2e/network/service_annotations.go

```diff
@@ -195,6 +196,28 @@ var _ = Describe("Service with annotation", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ret).To(Equal(true), "external ip %s is not in the target subnet %s", ip, newSubnetCIDR)
 	})
+	It("should bound to the specified load balancer from azure-load-balancer-mode [MultipleAgentPools][VMSS]", func() {
+		//get nodelist and providerID specific to an agentnodes
+		nodes, err := utils.GetAgentNodes(cs)
+		Expect(err).NotTo(HaveOccurred())
+		// Get all the vmss names from all node's providerIDs
+		vmss_name := sets.NewString()
+		for _, node := range nodes {
+			providerID := node.Spec.ProviderID
+			vmss, err := azure.ExtractScaleSetNameByProviderID(providerID)
+			Expect(err).NotTo(HaveOccurred())
+			vmss_name.Insert(vmss)
+		}
+		//Skip if there're less than two vmss nodes
```

Comment:
please use Skip() here.

```go
if len(vmss_name) < 2 {
  Skip("at least two vmss are required for azure-load-balancer-mode  validation")
}
```

Reply:
So, any line after Skip() outside the if condition will be executed only if the `vmss_name>2`